tutoraspects/plugin.py

```diff
@@ -86,6 +86,8 @@
             "* [Superset Resources](https://github.com/apache/superset#resources)<br>",
         ),
         ("ASPECTS_ENABLE_INSTRUCTOR_DASHBOARD_PLUGIN", True),
+        # Whether to show the link to go to Superset in the instructor dashboard tab
+        ("SUPERSET_SHOW_INSTRUCTOR_DASHBOARD_LINK", True),
```

Comment:
Do we need this setting? Operators can do the same using an inline plugin such as:

```yaml
name: aspects-settings
version: 0.1.0
patches:
    openedx-common-settings: |
      SUPERSET_SHOW_INSTRUCTOR_DASHBOARD_LINK = True
      # More aspects settings
```

Reply:
It would be weird that we set a bunch of settings for that plugin here, but not this one. I think it makes more sense to have there be defaults from the same place rather than have some missing.